tests/test_preprocessing.py

```diff
@@ -15,6 +15,7 @@
 
 import scanpy as sc
 from scanpy._compat import CSBase
+from scanpy.datasets._datasets import pbmc3k
```

Comment:
wrong import, please be careful that you import these from the testing helpers as intended